### PR TITLE
Updated param `location` to create a consistent description and updated schema references  

### PR DESCRIPTION
**Pull request by jasonviviano**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.dev/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Removed `location` if usage was `resourceGroup().location` or `global` and updated schema references.
Left `location` in snippets that specified locations like `westus2`.

## Issue reference
https://github.com/project-radius/docs/issues/544

### CODE REVIEW

File: docs/content/author-apps/cloud-resources/azure-resources/snippets/azure-connection.bicep [hunk 1]
```diff
@@ -1,11 +1,9 @@
 import radius as radius
 
 param environment string
-param location string = resourceGroup().location
 
 resource cache 'Microsoft.Cache/Redis@2019-07-01' = {
   name: 'mycache'
-  location: location
```
Comment (AaronCrawfis) [inline docs/content/author-apps/cloud-resources/azure-resources/snippets/azure-connection.bicep]: While Radius resources no longer need `location`, Azure resources still do. I'd keep all of the location params/properties for anything that's an Azure resource

File: docs/content/reference/resource-schema/core-schema/application-schema/_index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ weight: 100
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `frontend`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/application-schema/_index.md]: Good catch on these!

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Thanks for getting this in! 

Azure resources actually still need locations. Could you restore all the params/properties for Azure resources?

Markdown fixes were a good catch, thanks! Those we should definitely fix as part of this PR

File: docs/content/author-apps/cloud-resources/azure-resources/snippets/azure-connection.bicep (hunk 1 above)
Comment (jasonviviano) [reply docs/content/author-apps/cloud-resources/azure-resources/snippets/azure-connection.bicep]: Fixed! Added/changed old descriptions to the one used in the `location` param table

File: docs/content/author-apps/cloud-resources/azure-resources/snippets/azure-connection.bicep [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import radius as radius
 
 param environment string
+
+@description('The geo-location where the resource lives.')
```
Comment (AaronCrawfis) [inline docs/content/author-apps/cloud-resources/azure-resources/snippets/azure-connection.bicep]: Let's do something like:

```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

`geo-location` isn't a well-defined term.

File: docs/content/author-apps/links/snippets/redis-azure.bicep [hunk 4]
```diff
@@ -1,6 +1,8 @@
 import radius as radius
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/author-apps/links/snippets/redis-azure.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/concepts/links-concept/snippets/corerp-link-example.bicep [hunk 5]
```diff
@@ -2,6 +2,7 @@ import radius as radius
 
 param environment string
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/concepts/links-concept/snippets/corerp-link-example.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/getting-started/quickstarts/quickstart-container-wi/snippets/container-wi.bicep [hunk 6]
```diff
@@ -1,7 +1,7 @@
 //ENVIRONMENT
 import radius as radius
 
-@description('Specifies the location for Azure resources.')
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/getting-started/quickstarts/quickstart-container-wi/snippets/container-wi.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-dapr/snippets/dapr-azure.bicep [hunk 7]
```diff
@@ -1,6 +1,8 @@
 import radius as radius
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-dapr/snippets/dapr-azure.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-dapr/snippets/statestore-azure.bicep [hunk 8]
```diff
@@ -10,7 +10,9 @@ resource app 'Applications.Core/applications@2022-03-15-privatepreview' = {
 }
 
 //SAMPLE
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-dapr/snippets/statestore-azure.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-ui/snippets/dapr-azure.bicep [hunk 9]
```diff
@@ -1,6 +1,8 @@
 import radius as radius
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-ui/snippets/dapr-azure.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/getting-started/quickstarts/quickstart-keyvault-wi/snippets/keyvault-wi.bicep [hunk 10]
```diff
@@ -1,12 +1,12 @@
 //ENVIRONMENT
 import radius as rad
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/getting-started/quickstarts/quickstart-keyvault-wi/snippets/keyvault-wi.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/getting-started/supported-languages/bicep/bicep-infrastructure/snippets/new.bicep [hunk 11]
```diff
@@ -1,3 +1,4 @@
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/getting-started/supported-languages/bicep/bicep-infrastructure/snippets/new.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/operations/custom-recipes/snippets/recipe-params.bicep [hunk 12]
```diff
@@ -1,3 +1,4 @@
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/operations/custom-recipes/snippets/recipe-params.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/operations/custom-recipes/snippets/recipe.bicep [hunk 13]
```diff
@@ -1,9 +1,10 @@
-param location string = resourceGroup().location
-
 //RESOURCE
 @description('Radius-provided object containing information about the resouce calling the Recipe')
 param context object
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/operations/custom-recipes/snippets/recipe.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/reference/resource-schema/core-schema/volumes/volume-azure-keyvault/snippets/volume-keyvault.bicep [hunk 14]
```diff
@@ -1,5 +1,6 @@
 import radius as rad
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/reference/resource-schema/core-schema/volumes/volume-azure-keyvault/snippets/volume-keyvault.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-pubsub/snippets/dapr-pubsub-servicebus.bicep [hunk 15]
```diff
@@ -1,6 +1,8 @@
 import radius as radius
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-pubsub/snippets/dapr-pubsub-servicebus.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-statestore/snippets/dapr-statestore-sql.bicep [hunk 16]
```diff
@@ -1,6 +1,8 @@
 import radius as radius
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-statestore/snippets/dapr-statestore-sql.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-statestore/snippets/dapr-statestore-tablestorage.bicep [hunk 17]
```diff
@@ -1,6 +1,8 @@
 import radius as radius
 
+@description('The geo-location where the resource lives.')
```
Comment (jasonviviano) [inline docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-statestore/snippets/dapr-statestore-tablestorage.bicep]:
```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

File: docs/content/reference/resource-schema/core-schema/httproute/index.md [hunk 18]
```diff
@@ -15,7 +15,7 @@ weight: 400
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your HttpRoute. Used to provide status and visualize the resource. | `'web'`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/httproute/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

For now let's keep this as 'y', as it's required on-the-wire. We'll follow up with another PR updating this text with the nuance later.

File: docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-http/index.md [hunk 19]
```diff
@@ -22,7 +22,7 @@ A Dapr HTTP Route is defined as a resource within your Application, defined at t
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. Names must contain at most 63 characters, contain only lowercase alphanumeric characters, '-', or '.', start with an alphanumeric character, and end with an alphanumeric character. | `backend-route`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-http/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-pubsub/_index.md [hunk 20]
```diff
@@ -20,7 +20,7 @@ This resource will automatically create and deploy the Dapr component spec for t
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of the pub/sub. Names must contain at most 63 characters, contain only lowercase alphanumeric characters, '-', or '.', start with an alphanumeric character, and end with an alphanumeric character. | `my-pubsub` |
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-pubsub/_index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/link-schema/cache/redis/index.md [hunk 21]
```diff
@@ -26,7 +26,7 @@ The `redislabs.com/Redis` link is a [portable link]({{< ref links >}}) which can
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. | `cache`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/cache/redis/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-statestore/_index.md [hunk 22]
```diff
@@ -22,7 +22,7 @@ This resource will automatically create and deploy the Dapr component spec for t
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of the resource. Names must contain at most 63 characters, contain only lowercase alphanumeric characters, '-', or '.', start with an alphanumeric character, and end with an alphanumeric character. | `my-statestore` |
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-statestore/_index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/link-schema/messaging/rabbitmq/index.md [hunk 23]
```diff
@@ -22,7 +22,7 @@ The `rabbitmq.com/MessageQueue` link offers a [RabbitMQ message broker](https://
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. | `mongo`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/messaging/rabbitmq/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/link-schema/extender/_index.md [hunk 24]
```diff
@@ -20,7 +20,7 @@ An extender resource could be used to bring in a custom resource into Radius for
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. | `mongo`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/extender/_index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/link-schema/databases/mongodb/index.md [hunk 25]
```diff
@@ -26,7 +26,7 @@ The `mongodb.com/MongoDatabase` link is a [portable link]({{< ref links >}}) whi
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. | `mongo`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/databases/mongodb/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/link-schema/databases/microsoft-sql/index.md [hunk 26]
```diff
@@ -27,7 +27,7 @@ This application showcases how Radius can use a user-manged Azure SQL Database.
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. | `mongo`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/link-schema/databases/microsoft-sql/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/core-schema/gateway/index.md [hunk 27]
```diff
@@ -15,7 +15,7 @@ weight: 401
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your Gateway. | `'gateway'`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/gateway/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/core-schema/application-schema/_index.md (hunk 2 above)
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/application-schema/_index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/core-schema/container-schema/_index.md [hunk 28]
```diff
@@ -21,7 +21,7 @@ Containers are hosted by Kubernetes as the container runtime today, regardless o
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `frontend`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/container-schema/_index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

File: docs/content/reference/resource-schema/core-schema/environment-schema/index.md [hunk 29]
```diff
@@ -15,7 +15,7 @@ weight: 000
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | name | y | The name of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `frontend`
-| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
+| location | n | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```
Comment (AaronCrawfis) [inline docs/content/reference/resource-schema/core-schema/environment-schema/index.md]:
```suggestion
| location | y | The location of your resource. See [common values]({{< ref "resource-schema.md#common-values" >}}) for more information. | `global`
```

**Review by AaronCrawfis — APPROVED** [review]

**Review by kachawla — COMMENTED** [review]
:shipit:

**Pull request merged**